src/components/file/DropzoneHelper.tsx

```diff
@@ -0,0 +1,77 @@
+import { useContext, useRef } from 'react';
+
+import FolderOutlinedIcon from '@mui/icons-material/FolderOutlined';
+import UploadFileOutlinedIcon from '@mui/icons-material/UploadFileOutlined';
+import { Button, Stack, Typography } from '@mui/material';
+
+import { useBuilderTranslation } from '@/config/i18n';
+import { BUILDER } from '@/langs/constants';
+
+import { UppyContext } from './UppyContext';
+
+const DropzoneHelper = (): JSX.Element => {
+  const { t } = useBuilderTranslation();
+  const { uppy } = useContext(UppyContext);
+  const ref = useRef<HTMLInputElement>(null);
+
+  const handleClick = () => {
+    const { current } = ref;
+    if (current) {
+      current.click();
+    }
+  };
+
+  const handleFiles = () => {
+    const { current } = ref;
+    if (current) {
+      const { files } = current;
+      if (files) {
+        // add files selected to uppy, this will upload them
+        [...files].map((file) =>
+          uppy?.addFile({ data: file, name: file.name }),
+        );
+      } else {
+        console.error('no files found !');
+      }
+    }
+  };
+
+  return (
+    <Stack
+      justifyContent="flex-start"
+      alignItems="center"
+      direction="column"
+      flexGrow={1}
+      spacing={3}
+      sx={{ height: '100%', marginTop: '10%' }}
+    >
+      <UploadFileOutlinedIcon sx={{ fontSize: '50px', color: '#5050d2' }} />
+      <Typography variant="h4" sx={{ color: '#757575' }}>
+        {t(BUILDER.DROPZONE_HELPER_TEXT)}
+      </Typography>
+      <Typography variant="h5" sx={{ color: '#757575' }}>
+        {t('or')}
+      </Typography>
+      <Button
+        variant="contained"
+        size="large"
+        onClick={handleClick}
+        startIcon={<FolderOutlinedIcon />}
+      >
+        {t(BUILDER.DROPZONE_HELPER_ACTION)}
+      </Button>
+      <input
+        style={{ display: 'none' }}
+        type="file"
+        multiple
+        ref={ref}
+        onChange={handleFiles}
+      />
+      <Typography variant="h6" sx={{ color: '#757575' }}>
```

Comment:
Not sure we need to have a heading6 here, maybe use `variant="body1"` to make it more uniform with the rest of the interface.